docs/source/index.rst

```diff
@@ -42,7 +42,7 @@ Learn more about SuperstaQ `here <https://www.super.tech/about-superstaq/>`_.
 
    get_started/credentials
    get_started/installation
-   get_started/basics
+   get_started/basics/basics
```

Comment:
`basics/basics`

Redundant

Reply:
The `basics.rst` file is in the `basics` directory. Unless you're referring to something else?

Reply:
Ah okay. Sounds good.